src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift

```diff
@@ -38,7 +38,7 @@ struct ActiveTracing: Equatable {
 		// Hours are intentionally rounded down.
 		// We could also simply cast this to `Int` (what we actually do here as well)
 		// but we still call rounded(…) to make it more explicit.
-		Int((interval / TimeInterval.SEC_PER_HOUR).rounded(.down))
+		return 24 * 14 //Int((interval / TimeInterval.SEC_PER_HOUR).rounded(.down))
```

Comment:
I guess you intentionally left this comment in, right?

Reply:
Thanks! Removed debugging code.